pkgs/applications/emulators/dolphin-emu/default.nix

```diff
@@ -124,6 +117,8 @@ stdenv.mkDerivation rec {
     sfml
     xxHash
     xz # LibLZMA
+    # FIXME: Causes linker errors
+    #zlib-ng
```

Comment:
im guessing this is meant to be used over a submodule. might be good to specify that along with why it doesn't work. otherwise lgtm